google_shopping.js

```diff
@@ -1,5 +1,45 @@
 var data = require("./products.json")
 
 // Write your solutions below
+1. 
+var count = 0
+for (var i = 0; i < data["items"].length; i++) {
+	if (data["items"][i].kind === "shopping#product") {
+		count++;
+	}
+} console.log(count);
+// 25 items with shopping#product in "kind"
 
-console.log(data["items"]);
\ No newline at end of file
+2.
+for (var i = 0; i < data["items"].length; i++) {
+	if (data.items[i].product.inventories[0].availability == "backorder") {
```

Comment:
Fantastic job on the HW! Very clean code! A couple notes:

- For question 2, notice that you're using the [0] index of 'inventories'. Think about how you could make this for loop more extensible so that even if 'inventories' had more than one entry, you could still find all the backorder entries.

- Make sure to set your Sublime settings to have tabs set to 2 and convert tabs to spaces. Increases readability

Keep it up!